servers/parser-graphql-wrapper/src/marticle/marticleParser.ts

```diff
@@ -347,9 +348,31 @@ function RootNode(input: string) {
     '<x-marticle id="marticle-root">' + input + '</x-marticle>',
     null,
   );
+  // Remove any blank or whitespace nodes from the body
+  removeWhitespaceNodes(doc.body);
```

Comment:
We were trimming whitespace in our tests, so instead, this moves it to after we parse the dom.